### PR TITLE
Flz 4218 download model

### PR DESCRIPTION
**Pull request by danaturjeman-folloze**

### CODE REVIEW

**Comment by TanyaFolloze** [comment]
Task linked: [FLZ-4218 Download Model](https://app.clickup.com/t/3749831/FLZ-4218)

File: src/liveboard/Liveboard.ts [hunk 1]
```diff
@@ -285,10 +285,22 @@ export class Liveboard {
      * @param {number} itemId
      * @returns {ItemDownloadUrlSuccessResponseV2|ItemDownloadUrlFailedResponseV2} the url or failiure message
      */
-    getItemDownloadUrl(itemId: number): Promise<ItemDownloadUrlSuccessResponseV2 | ItemDownloadUrlFailedResponseV2> {
+    getItemDownloadUrl(
+        sourceType: string,
+        boardId: number,
+        contentItemId: number,
+        itemId: number
```
Comment (rotemgrim) [inline src/liveboard/Liveboard.ts:292]: why you need all this params?
isn't contentItemId not enough?

**Comment by rotemgrim** [comment]
and you got a conflict to solve here

**Review by rotemgrim — CHANGES_REQUESTED** [review]

File: src/liveboard/Liveboard.ts (hunk 1 above)
Comment (danaturjeman-folloze) [reply src/liveboard/Liveboard.ts:292]: currently, the lead is able to perform actions only on items, and then item_id was enough.

after recommendations and content containers will be applied, the lead will be able to perform actions on content items as well and then item_id will be assigned to null and sourceType will indicate whether it's AI, ContentContainer, etc.

anyway, when the sourceType is Item, the itemId attribute is necessary for analytics purpose.

**Review by rotemgrim — CHANGES_REQUESTED** [review]

**Review by rotemgrim — APPROVED** [review]

**Pull request merged**